package.json

```diff
@@ -21,8 +21,7 @@
     "start-dev": "nodemon --inspect server.js",
     "prestart-dev": "npm run lint",
     "migrations": "sequelize db:migrate",
-    "start": "node server.js",
-    "server": "babel-node server.js"
+    "start": "babel-node --inspect server.js"
```

Comment:
Why do you need babel to start the app?

Reply:
It wasn't detecting some of the nicer syntax from ES6 If I remember right and it was throwing errors.